v03_pipeline/lib/misc/validation.py

```diff
@@ -13,14 +13,17 @@ class SeqrValidationError(Exception):
 
 def validate_allele_type(
     mt: hl.MatrixTable,
+    dataset_type: DatasetType,
 ) -> None:
     ht = mt.rows()
     ht = ht.filter(
-        hl.numeric_allele_type(ht.alleles[0], ht.alleles[1])
-        == hl.genetics.allele_type.AlleleType.UNKNOWN,
+        dataset_type.invalid_allele_types.contains(
```

Comment:
I fixed this to be more precise in the error message... using a collected set rather than just the first 10 alleles.